py/persistentcode.c

```diff
@@ -633,6 +636,7 @@ STATIC void save_raw_code(mp_print_t *print, mp_raw_code_t *rc, qstr_window_t *q
         // Save native code
```

Comment:
This entire "else" block can be guarded by `MICROPY_EMIT_NATIVE || MICROPY_EMIT_INLINE_ASM`, instead of the two separate `#if`'s below.

Reply:
Makes sense. I'll update the reader part too.